### DISCUSSION TITLE
School Management System

### DISCUSSION THREAD

**Discussion by Saadie1Asad** [community/community#24136] [GitHub Education]
<p>Hi Friends,</p>
<p>Please I am looking for school Management system, if some has please let me know.</p>
<p>thank you</p>
<p>Regards,</p>
<p>Asad</p>

**Comment by mpboom** [comment]
<p>Please clarify what it exactly is that you are looking for.</p>

**Comment by Saadie1Asad** [comment]
<p>thank you for reply, I am looking for students registration , attendence , exam etc </p>

**Comment by Hc747** [comment]
<p>This might be close to what you’re looking for: <a href="https://www.schoolbytes.education/" rel="nofollow noopener">https://www.schoolbytes.education/</a></p>

**Comment by Saadie1Asad** [comment]
<p>thank you for reply, please how I can download available on <a href="http://github.com" rel="nofollow noopener">github.com</a> </p>

**Comment by lakshitkale** [comment]
<p>Just go through this link and you will definitely get your solution. <a href="https://www.iitms.co.in/school-erp" rel="nofollow noopener">https://www.iitms.co.in/school-erp</a></p>

**Comment by fire-eggs** [comment, marked as answer]
<p>Go to <a href="http://github.com" rel="nofollow noopener">github.com</a>. In the upper left corner of the page is a search box. Typing in “school management” returns 5,123 repositories.</p>

**Comment by jamescaan20** [comment]
<p>There are following School management system with tons of features …</p>
<ul>
<li><a href="https://www.powerschool.com/" rel="nofollow noopener">https://www.powerschool.com/</a></li>
<li>
</li>
</ul>
<p><a href="https://classroom.google.com" rel="nofollow noopener">https://classroom.google.com</a></p>
<ul>
<li><a href="https://www.edusys.co/en-in/school-management-software.html" rel="nofollow noopener">https://www.edusys.co/…/school-management-software.html</a></li>
<li><a href="https://moodle.org/" rel="nofollow noopener">https://moodle.org/</a></li>
<li><a href="http://www.koha.org/" rel="nofollow noopener">http://www.koha.org/</a></li>
<li>…</li>
</ul>

**Comment by ds12344** [comment]
<p>Hi I am looking for SRS of school management system. Can you help me with that please?</p>

**Comment by rashidkhan98** [comment]
<p>Hi,<br>
My name is rashid and currently i am using IQminds school manage system software it’s available at affordable price with useful features like managing students, attendance, exam, hr payroll, teacher and staff etc. Contact me or visit the link to get the best <a href="https://www.iqminds.com/school-college-management-software-uae/" rel="nofollow noopener">school management system</a> software in dubai UAE.</p>

**Comment by soniyabawane** [comment]
<p>Hello , we have a School magica -<a href="https://www.schoolmagica.com/" rel="noopener nofollow ugc">School management software</a> .With this software you can manage attendace , fees student , class , staff.</p>

**Comment by alli-invent** [comment]
> Hi Friends,
> 
> Please I am looking for school Management system, if some has please let me know.
> 
> thank you
> 
> Regards,
> 
> Asad

I do not know if you have gotten a school management system. I do recommend the FREE Plan from newbrandtechnologies.com. It has so many premium features for FREE. The free plan has all what a school need to run. Check it!
